docs/advanced-techniques.md

```diff
@@ -88,3 +88,27 @@ A module is a standalone configuration object that can reference client code, re
 A merged module's code is treated the same as the application's code. The resulting application's bundles include the module code and configurations as if they were always a part of the application.
```

Comment:
This is a strange paragraph. "is treated the same"? " as if they were always a part"?

Reply:
Rewrote the paragraph like so:
"A merged module's code is included into the resulting application's bundles and behaves as a part of the application."